keyboards/melgeek/z70ultra/z70ultra.c

```diff
@@ -0,0 +1,166 @@
+	{
+		/* C0     C1      C2      C3      C4      C5      C6      C7      C8      C9       C10      C11     C12     C13      */
+		{  0,      1,      2,      3,     4,       5,      6,      7,      8,      9,       10,      11,     12,     13,     }, /* R0 */
+		{ 15,     16,     17,     18,     19,     20,     21,     22,     23,     24,       25,      26,     27,     28,     }, /* R1 */
+		{ 30,     31,     32,     33,     34,     35,     36,     37,     38,     39,       40,      41,     NO_LED, 42,     }, /* R2 */
+		{ 44,     NO_LED, 45,     46,     47,     48,     49,     50,     51,     52,       53,      54,     55,     NO_LED, }, /* R3 */
+		{ 58,     59,     60,     NO_LED, 61,     62,     63,     NO_LED, NO_LED, 64,       65,      NO_LED, NO_LED, NO_LED, }, /* R4 */
+		{ NO_LED, NO_LED, NO_LED, NO_LED, NO_LED, NO_LED, 14,     29,     56,     43,       57,      66,     67,     68,     }, /* R5 */
+    }, {
+        {0,  0},  {16,  0},  {32,  0}, {48,  0}, {64,  0}, {80,  0}, {96,  0}, {112,  0}, {128,  0}, {144,  0}, {160,  0}, {176,  0}, {192,  0}, {208,  0}, 
+		{0, 16},  {16, 16},  {32, 16}, {48, 16}, {64, 16}, {80, 16}, {96, 16}, {112, 16}, {128, 16}, {144, 16}, {160, 16}, {176, 16}, {192, 16}, {208, 16}, 
+		{0, 32},  {16, 32},  {32, 32}, {48, 32}, {64, 32}, {80, 32}, {96, 32}, {112, 32}, {128, 32}, {144, 32}, {160, 32}, {176, 32},            {192, 32}, 
+		{0, 48},             {16, 48}, {32, 48}, {48, 48}, {64, 48}, {80, 48}, {96, 48},  {112, 48}, {128, 48}, {144, 48}, {160, 48}, {176, 48},
+		{0, 64},  {16, 64},  {32, 64},           {48, 64}, {64, 64}, {80, 64},                        {96, 64}, {112, 64}, 
+		                                                             {0, 80},  {16, 80},  {32, 80},   {48, 80}, {64, 80}, {80, 80},   {96, 80}, {112, 80}, 
+	}, {
+	       4,       4,         4,        4,        4,        4,        4,        4,          4,         4,         4,          4,        4,         4,
+		   4,       4,         4,        4,        4,        4,        4,        4,          4,         4,         4,          4,        4,         4,
+		   4,       4,         4,        4,        4,        4,        4,        4,          4,         4,         4,          4,                   4,
+		   4,                  4,        4,        4,        4,        4,        4,          4,         4,         4,          4,        4,
+		   4,       4,         4,                  4,        4,        4,                               4,         4,
+		                                                               4,        4,          4,         4,         4,          4,        4,         4,
+	}
+};
+
+const is31_led g_is31_indicator_leds[6] = {
+    {0, CS5_SW1, CS3_SW1, CS4_SW1}, /* RGB71 */
+    {0, CS5_SW2, CS3_SW2, CS4_SW2}, /* RGB72 */
+    {0, CS5_SW3, CS3_SW3, CS4_SW3}, /* RGB73 */
+    {0, CS5_SW4, CS3_SW4, CS4_SW4}, /* RGB74 */
+    {0, CS5_SW5, CS3_SW5, CS4_SW5}, /* RGB75 */
+    {0, CS5_SW6, CS3_SW6, CS4_SW6}, /* RGB76 */
+};
```

Comment:
This shouldn't be needed anymore, actually. 

you can use `8` for the flags for the LEDs in the led config structure, and then handle that based on that.

Reply:
It took a few moments to interpret that number `8` for me.
https://github.com/qmk/qmk_firmware/blob/0b2eae768637bc9230fec5d5167f617b21f4e179/quantum/rgb_matrix_types.h#L56-L59
https://github.com/qmk/qmk_firmware/blob/0b2eae768637bc9230fec5d5167f617b21f4e179/keyboards/planck/ez/ez.c#L102-L105
It seems no keyboard is using this value, `LED_FLAG_INDICATOR` .
I'd rather inclined to use `0` instead. In hope of keeping those leds untouched by any other code that exist in qmk.

So re-thinking with these concerns in mind, controlling indicator leds at lower level using led-driver interface is a reasonable design choice. I might say.